docs/core/macos-prerequisites.md

```diff
@@ -17,7 +17,7 @@ This article shows you the supported macOS versions and .NET Core dependencies t
 
```

Comment:
can you also update the ms.date to 10/03/2018?

Reply:
You beat me to it ...